pkg/loki/loki.go

```diff
@@ -674,13 +674,25 @@ func (t *Loki) setupModuleManager() error {
 		mm.RegisterModule(QueryLimitsInterceptors, t.initQueryLimitsInterceptors, modules.UserInvisibleModule)
 		mm.RegisterModule(QueryLimitsTripperware, t.initQueryLimitsTripperware, modules.UserInvisibleModule)
 
-		deps[Querier] = append(deps[Querier], QueryLimiter)
-		deps[QueryFrontend] = append(deps[QueryFrontend], QueryLimitsTripperware)
-
+		// Ensure query limiter embeds overrides after they've been
+		// created.
 		deps[QueryLimiter] = []string{Overrides}
 		deps[QueryLimitsInterceptors] = []string{}
+
+		// Ensure query limits tripperware embeds the query frontend
+		// tripperware after it's been created.
```

Comment:
can we include in this comment `any additional milleware/tripperware you want to add to the querier or frontend must happen after this line`? I think this is the right spot for that addition